module/apps.js

```diff
@@ -131,6 +145,7 @@ class GroupRollApp extends Application {
                 icon: "fas fa-undo",
                 onclick: ev => {
                     canvas.tokens.ownedTokens.map(t => this.mstList[t.id] = { adv: 0, bon: 0, roll: { total: "", result: "", terms: [{ total: 10 }] } });
+                    this.commitValues();
```

Comment:
Please delete this line.  This prevents the Reset button from resetting the bonus values in the form.

Reply:
61ac3424a1d5f24deb831e6e5fa167b30f3507a3